include/deal.II/matrix_free/evaluation_kernels.h

```diff
@@ -2652,6 +2652,8 @@ namespace internal
                  orientation[v][i];
       };
 
+      bool writes_into_vector = true;
```

Comment:
May I ask you to add a comment here.

Reply:
I added a comment here and further down where the variable is set (or the early return for the read path).